accounts/models.py

```diff
@@ -536,7 +580,6 @@ class Message(models.Model):
     def send_as_email(self):
         context = {
             "base_url": settings.BASE_URL,
-            "osf_url": settings.OSF_URL,
             "custom_message": mark_safe(self.body),
```

Comment:
(I'm not saying much because everything is looking really nice and clearly written, but I am at least double-checking e.g. to make sure there aren't any uses of `osf_url` as I go along)